mappings/net/minecraft/block/Material.mapping

```diff
@@ -4,45 +4,45 @@ CLASS net/minecraft/class_3614 net/minecraft/block/Material
 	FIELD field_15913 COBWEB Lnet/minecraft/class_3614;
 	FIELD field_15914 STONE Lnet/minecraft/class_3614;
 	FIELD field_15915 BUBBLE_COLUMN Lnet/minecraft/class_3614;
-	FIELD field_15916 SAND Lnet/minecraft/class_3614;
+	FIELD field_15916 AGGREGATE Lnet/minecraft/class_3614;
```

Comment:
This sounds a bit weird. It's used for powder-like falling blocks (sands, soul sand, gravel and concrete powder), so maybe `POWDER`?

Reply:
or `FALLABLE`?

Reply:
I prefer `AGGREGATE` over `POWDER`. `FALLABLE` is not in keeping with the rest of the materials being some descriptor of what the blocks are/are made from

Reply:
_Aggregate_ is vaguely familliar. Looking it up, I get the definition "a natural mixture of sand, gravel, and crushed stone" in the context of geology, though I agree it's a little too technical. If you use it, this will require a Javadoc.

Reply:
![image](https://user-images.githubusercontent.com/6429283/76984917-2787e280-6948-11ea-9867-b17ad1ea4a36.png)
> https://en.wikipedia.org/wiki/Aggregate_(geology)

Reply:
Aggregate is a fairly common term. Regardless, it is more descriptive of these blocks than "sand" or "powder."

Reply:
"Particles" is used elsewhere by the game, and you can't really call gravel particles. "Aggregate" is more descriptive.